SPDX-FileCopyrightText: 2022 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: PSF-2.0

Again, Python 2.5 added the key= argument for sorting functions,
which is not available when we are building Python 2.5.

Sorting is absolutely unnessecary when generating defines for a
header file so we can just remove it.

--- Lib/sre_constants.py	2004-08-25 12:22:30.000000000 +1000
+++ Lib/sre_constants.py	2022-10-09 20:18:40.332233858 +1100
@@ -219,7 +219,6 @@
 if __name__ == "__main__":
     def dump(f, d, prefix):
         items = d.items()
-        items.sort(key=lambda a: a[1])
         for k, v in items:
             f.write("#define %s_%s %s\n" % (prefix, k.upper(), v))
     f = open("sre_constants.h", "w")
